### PR TITLE
Like count testing

### PR DESCRIPTION
**Pull request by SekoViper**
### Implemented the following: 

- Items counter is covered with unit tests using Jest
- Add a mock function in the __mock__ folder
- Updated the API credentials for a specific file.

### CODE REVIEW

File: __mocks__/getLikesMock.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
```
Comment (mkaya13) [inline __mocks__/getLikesMock.js:1]: It's good that you used axios which works built-in with jest framework!

File: __mocks__/getLikesMock.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+export const getLikesMock = async (API_PATH) => {
```
Comment (mkaya13) [inline __mocks__/getLikesMock.js:3]: Mocks are essential for integration tests. So its good that you wrote a mock function.

File: __mocks__/jestAPICredentials.js [hunk 3]
```diff
@@ -2,6 +2,8 @@ export const jestAPICredentials = {
   BASE_COMMENTS_API:
     'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/',
   COMMENTS_ENDPOINT_ID: 'QhwyHRqxtZN5QJVQP2gT',
+  LIKES_ENDPOINT_API:
```
Comment (mkaya13) [inline __mocks__/jestAPICredentials.js:5]: As we talked previously, we were not able to extract credentials data from .env file while using jest. So we add credentials over here for testing. We are going to hide that file when we are done with everything.

File: tests/getLikesMock.test.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+// arrange
+describe('Tests fetch with axios', () => {
+  const { LIKES_ENDPOINT_API } = jestAPICredentials;
+  const endpoint = LIKES_ENDPOINT_API;
+
+  it('Grabs Likes Count', async () => {
+    const likesData = await getLikesMock(endpoint);
+    let likeCount = '';
+    likesData.forEach((likeDataObject) => {
```
Comment (mkaya13) [inline tests/getLikesMock.test.js:12]: It's nice idea to run a forEach function to get the like count of specific item inside the API endpoint.

**Review by mkaya13 — COMMENTED** [review]
Hi Seko, 

Good job so far!
You are good to go everything is set and working!

## Highlights
- Linters are passing
- Tests are working

## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: __mocks__/getLikesMock.js (hunk 1 above)
Comment (SekoViper) [reply __mocks__/getLikesMock.js:1]: Thanks

File: __mocks__/getLikesMock.js (hunk 2 above)
Comment (SekoViper) [reply __mocks__/getLikesMock.js:3]: Thanks

**Review by omar-muhamad — COMMENTED** [review]
# Invalid ❌

Hi @SekoViper and @mkaya13,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Your Code Review Request is marked as invalid for these reasons:

- [ ] You didn't follow the correct Gitflow workflow, Please make this Pull Request from the development(dev) branch to the main branch to make it contains all changes you have done inside this project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Pull request merged**